w3lib/encoding.py

```diff
@@ -22,14 +22,25 @@ def http_content_type_encoding(content_type):
 
 # regexp for parsing HTTP meta tags
 _TEMPLATE = r'''%s\s*=\s*["']?\s*%s\s*["']?'''
+_SKIP_ATTRS = r'''(?x)(?:\s+
+    [^=<>/\s"']+  # Attribute name
```

Comment:
For completeness we can also exclude non-printable characters from HTML attribute name regex.